src/routes/routes.ts

```diff
@@ -9,7 +9,8 @@ import Profile from "../pages/Profile";
 import FAQ from "../pages/FAQ";
 import JoinUs from "../pages/JoinUs";
 import Donate from "../pages/Donate";
-import NeedsView from "../components/needLeadViewPage/needsView"
+import NeedsView from "../components/needLeadViewPage/needsView";
+import LeadsView from "../components/needLeadViewPage/leadsView";
 
```

Comment:
if those components are Pages, they should be in  the /pages directory and not /component :)

Reply:
ok, i will move it !